src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -287,7 +293,44 @@ limitations under the License.
                 ptype="LAG_ID"
                 >
         </PARAM>
-        <ACTION builtin="clish_restcl">oper=PATCH url=/restconf/data/openconfig-interfaces:interfaces/interface=PortChannel${lag-id}/config body={"openconfig-interfaces:config": {"name": "PortChannel${lag-id}"}}</ACTION>
+        <PARAM 
+                name="mode" 
+                help="PortChannel Mode" 
+                ptype="SUBCOMMAND" 
+                mode="subcommand" 
+                optional="true"
+                >
+                <PARAM 
+                        name="PoMode" 
+                        help="" 
+                        ptype="MODE"
+                        > 
+                </PARAM>
```

Comment:
plz remove the mode option from line #472, since we are already setting during create.

Reply:
Done